webroot/js/generic_functions.js

```diff
@@ -31,6 +31,11 @@ function get_tatoeba_root_url() {
     return "//" + host + "/"+ interfaceLang;
 }
 
+function get_csrf_token() {
+    // Probably not worth importing jquery.cookie for this.
+    return document.cookie.replace(/^.*csrfToken=([^;]*);.*$/, "$1");
```

Comment:
Just one small issue on my side: this regex won't work if the csrfToken is defined at the end of `document.cookie`. 

For instance my cookie looked like this:
`CakeCookie=%7B%22interfaceLanguage%22%3A%22eng%22%7D; csrfToken=8ccb558280c7ad3e0729dcbf0720fd78c23305a9b559877fe8ab534216f482004e9e6991e425155aec6f5f44a4617a633501b005ee354a7a26755367f52bb015`

and therefore the function returned the whole string.

Reply:
Good that you caught this. I hadn't considered that there's no semicolon after the value if it's the last entry. Should I just include `jquery.cookie` on all pages? It's 3121 bytes, so probably won't noticeably slow things down.

Reply:
Eventually we should try to remove jQuery (moving everything to angularjs), so it's probably best not to include jquery.cookie.

I would say just reuse one of the solutions suggested on Stack Overflow:
https://stackoverflow.com/questions/10730362/get-cookie-by-name
The most upvoted answer should be fine.

Reply:
Done. I wasn't sure about the legality of including a CC BY-SA 4.0 StackOverflow answer in an AGPL codebase, but CC BY-SA 4.0 is declared compatible with inclusion in GPLv3 and GPLv3 allows combining with AGPL, so this should be fine.

Reply:
Thanks :)